components/footer/index.scss

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) IBM, Corp. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+@import "@carbon/colors/scss/colors";
+@import "@carbon/layout/scss/layout";
+@import "@carbon/type/scss/type";
+
+:host {
+  margin: 0;
+}
+
+footer {
+  display: block;
+}
+
+.secondary-footer {
+  display: block;
+  padding: 2rem 0 4.5rem;
+}
+
+.main-footer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 2rem 0;
+  padding-right: 2.5rem;
+  padding-left: 2.5rem;
+  background-color: #f2f4f8;
+}
+
+.app-logo {
+  flex: 1.5 150px;
+  flex-basis: 0;
+  margin-right: auto;
+  font-family: carbon--font-family("sans");
+  fill: #697077;
+
+  svg {
+    height: 32px;
+  }
+
+  path {
+    background-color: #697077;
+    color: #697077;
+    fill: #697077;
+  }
+}
+
+.footer-section-title {
+  display: block;
+  margin-bottom: 1.5rem;
+  color: #697077;
+  font-weight: 600;
+  font-size: 0.875rem;
+  font-family: "IBM Plex Sans SemiBold", sans-serif;
+  line-height: 1.25;
+  letter-spacing: 0.16px;
+}
+
+li {
+  list-style: none;
+}
+
+ul {
+  padding-left: 0;
+}
+
+.link {
+  display: inline-block;
+  width: 100%;
+  padding-bottom: 0.5rem;
+  color: #697077;
+  font-weight: 400;
+  font-size: 0.75rem;
+  line-height: 1.3333;
+  letter-spacing: 0.32px;
```

Comment:
We use to use [carbon type includes](https://carbondesignsystem.com/guidelines/typography/type-sets): `@include carbon--type-style('body-long-01');` If I remember well `'body-long-01'` is the one that works here, but you can check it.

Reply:
Done!